virttest/libvirt_storage.py

```diff
@@ -281,12 +281,12 @@ def set_pool_autostart(self, name, extra=""):
         logging.info("Set pool '%s' autostart.", name)
         return True
 
-    def build_pool(self, name):
+    def build_pool(self, name, options=""):
```

Comment:
even `**dargs` can be included and sent to `virsh.pool_build` which make it more generic